packages/components/src/composite/stories/index.story.tsx

```diff
@@ -0,0 +1,218 @@
+import { UseCompositeStorePlaceholder, transform } from './utils';
+
+const meta: Meta< typeof UseCompositeStorePlaceholder > = {
+	title: 'Components/Composite',
+	component: UseCompositeStorePlaceholder,
+	subcomponents: {
+		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
+		Composite,
+		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
+		'Composite.Group': Composite.Group,
+		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
+		'Composite.GroupLabel': Composite.GroupLabel,
+		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
+		'Composite.Row': Composite.Row,
+		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
+		'Composite.Item': Composite.Item,
+	},
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			source: { transform },
+			extractArgTypes: ( component: React.FunctionComponent ) => {
+				const commonArgTypes = {
+					render: {
+						name: 'render',
+						description:
+							'Allows the component to be rendered as a different HTML element or React component. The value can be a React element or a function that takes in the original component props and gives back a React element with the props merged.',
+						table: {
+							type: {
+								summary:
+									'RenderProp<React.HTMLAttributes<any> & { ref?: React.Ref<any> | undefined; }> | React.ReactElement<any, string | React.JSXElementConstructor<any>>',
+							},
+						},
+						// type: { required: true },
+					},
+					children: {
+						name: 'children',
+						description: 'The contents of the component.',
+						table: { type: { summary: 'React.ReactNode' } },
+					},
+				};
+				const argTypes = {
+					useCompositeStore: {
+						activeId: {
+							name: 'activeId',
+							description:
+								'The current active item id. The active item is the element within the composite widget that has either DOM or virtual focus.',
+							table: { type: { summary: 'string | null' } },
+						},
+						defaultActiveId: {
+							name: 'defaultActiveId',
+							description:
+								'The composite item id that should be active by default when the composite widget is rendered. If `null`, the composite element itself will have focus and users will be able to navigate to it using arrow keys. If `undefined`, the first enabled item will be focused.',
+							table: { type: { summary: 'string | null' } },
+						},
+						setActiveId: {
+							name: 'setActiveId',
+							description:
+								'A callback that gets called when the activeId state changes.',
+							table: {
+								type: {
+									summary:
+										'((activeId: string | null | undefined) => void)',
+								},
+							},
+						},
+						focusLoop: {
+							name: 'focusLoop',
+							description:
+								'Determines how the focus behaves when the user reaches the end of the composite widget.',
```

Comment:
The original [ariakit docs](https://ariakit.org/reference/use-composite-store#focusloop) have a lot more info on some of these props, but I wasn't sure about adding all of that info here — partially because in Storybook it wouldn't be formatted nicely (including hyperlinks), and partially because it references other props that we are not exposing.